packages/aws-cdk-lib/assertions/test/annotations.test.ts

```diff
@@ -209,7 +209,7 @@ class MyAspect implements IAspect {
   };
 
   protected warn(node: IConstruct, message: string): void {
-    Annotations.of(node).addWarning(message);
+    Annotations.of(node).addWarningV2(node.node.id, message);
```

Comment:
I know this is only for tests, but a node ID as warning doesn't make sense to me. You could have put a dummy string here.